Create-ClientSecrets.ps1

```diff
@@ -1,6 +1,6 @@
 param(
     [Parameter()][string]$SettingsFile = 'developer-settings.json',
-    [Parameter()][string]$EnvironmentName = $SettingsFile.ResourceGroupName
+    [Parameter(Mandatory)][string]$EnvironmentName
```

Comment:
I assume you refer to this https://github.com/by-pinja/Protacon.RxMq/commit/86cb2de28919d29e73ab92fc16e064db869616ae

Yes, removing `$EnvironmentName` would/will break CI build, because removing parameter that is in use causes error. We can't use `$SettingsFile.ResourceGroupName` because ` $SetttingsFile` is string, and not yet parsed as JSON. I assume no-one actually tried to execute this script with the parameter actually missing.

By making this `Mandatory` we are forcing user to actually give this instead of failing because it's missing. CI is not affected because CI uses `-EnvironmentName` to give this.